app/views/idv/phone_errors/warning.html.erb

```diff
@@ -16,10 +16,12 @@
       ].select(&:present?),
     ) do %>
 
-      <p>
-        <%= t('idv.failure.phone.warning.you_entered') %>
-        <strong><%= PhoneFormatter.format(@phone, country_code: @country_code) %></strong>
-      </p>
+      <% if @phone %>
```

Comment:
Looks fine for now, could add a before action to guard against getting here without a phone.